server/app/middlewares/websocket_backend.rb

```diff
@@ -237,23 +237,22 @@ def subscribe_to_rpc_channel
 
   # @param [Hash] msg
   def on_rpc_message(msg)
-    client = client_for_id(msg['id'])
-    if client
-      self.send_message(client[:ws], msg['message'])
-    end
+    EM.next_tick{
+      client = client_for_id(msg['id'])
+      if client
+        self.send_message(client[:ws], msg['message'])
+      end
+    }
   rescue => exc
```

Comment:
Letting the `EM.next_tick { ... }` raise seems to cause weird things (the `MongoPubsub` actor dies without any errors logged)... move the `EM.next_tick{ ... }` into the `subscribe_to_rpc_channel` block, or move this rescue into the `EM.next_tick` block.